src/Microsoft.DotNet.ProjectModel/Graph/LockFileTargetLibrary.cs

```diff
@@ -34,5 +35,50 @@ public class LockFileTargetLibrary
         public IList<LockFileContentFile> ContentFiles { get; set; } = new List<LockFileContentFile>();
 
         public IList<LockFileRuntimeTarget> RuntimeTargets { get; set; } = new List<LockFileRuntimeTarget>();
+
+        public void MergeWith(LockFileTargetLibrary m)
+        {
+            TargetFramework = TargetFramework ?? m.TargetFramework;
+
+            if (Dependencies == null || !Dependencies.Any())
```

Comment:
@davidfowl @abpiskunov 

Should I union these, blindly overwrite, or keep them as is?